src/pages/signup-affiliates/components/_signup-form.tsx

```diff
@@ -172,6 +173,8 @@ const AffiliateSignupForm = ({
                     secondary
                     onClick={() => {
                         window.scrollTo(0, 0)
+                        console.log(affiliate_account.email)
```

Comment:
Do we need this console log @NikitK-deriv ? 🤔

Reply:
nope, I already removed it :)